config/opensearch_dashboards.yml

```diff
@@ -87,10 +87,10 @@
 
 # List of OpenSearch Dashboards client-side headers to send to OpenSearch. To send *no* client-side
 # headers, set this value to [] (an empty list).
-#opensearch.requestHeadersWhitelist: [ authorization ]
+#opensearch.requestHeadersAllowlist: [ authorization ]
```

Comment:
Should we be keeping the existing config to preserve backwards compatibility?

Reply:
I can create a separate entry in the yml file and just put deprecated for the old one but regardless of the example we have commented there it will work with either config name